ex_schemas/schema_v3.json

```diff
@@ -295,6 +295,9 @@
                     "type":"string",
                     "pattern":"libvirt_network"
                 },
+                "driver": {
```

Comment:
I guess main idea is to give users an option to chose their type drive which can be either "kvm" or qemu right ?
I that case , it would be great if we can limit their options in schema by 
 "driver": {
             "enum": ["kvm", "qemu"]
           }
}
Since we are planning to rewrite the schema validation , the existing also works.